docs/NODE_FORMAT.md

```diff
@@ -0,0 +1,36 @@
+## Corfu Node IDs
+
+This document describes the format of Corfu node identifiers (node IDs), which are used to refer to unique Corfu instances.
+Node Ids provide an extra layer of reliability in the case of a server or network reconfiguration. For example, a DNS error
+could point a Corfu client at an incorrect node. This could lead a client to read an update from the wrong system, or believe
+that it has committed an update when it is not committed. Node IDs prevent this issue by assigning each node a unique 128-bit
+identifier at startup. When clients connect to a Corfu node, they verify that the correct Node ID is presented by the node
+before interacting with it.
+
+Node IDs consist of the following components:
+
+- ``<protocol>``: The protocol that used by the node. Currently, only ``tcp`` is supported.
```

Comment:
that is used... or: The protocol used by...

Reply:
thanks@